compiler-rt/lib/tsan/rtl/tsan_report.cpp

```diff
@@ -103,12 +103,22 @@ void PrintStack(const ReportStack *ent) {
     Printf("    [failed to restore the stack]\n\n");
     return;
   }
+
+  if (const ListOfModules *modules =
+          Symbolizer::GetOrInit()->GetRefreshedListOfModules()) {
+    InternalScopedString modules_res;
+    RenderModules(&modules_res, modules,
```

Comment:
I would prefer if this and the other caller looked like

```
if (common_flags()->enable_symbolizer_markup) 
   RenderModulesMarkup(...);
```

The indirection doesn't add much and only obscures what is happening.